site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1218,6 +1224,8 @@ public function renderRubricPanel(GradedGradeable $graded_gradeable, int $displa
 
         return $return . $this->core->getOutput()->renderTwigTemplate("grading/electronic/RubricPanel.twig", [
                 "showNewInterface" => $showNewInterface,
+                "gradeable" => $gradeable,
+                "anon_ids" => $anon_ids,
                 "gradeable_id" => $gradeable->getId(),
                 "is_ta_grading" => $gradeable->isTaGrading(),
                 "anon_id" => $graded_gradeable->getSubmitter()->getAnonId(),
```

Comment:
I see that there is both an `anon_id` and an `anon_ids` here. Do we need both?

Reply:
Yes, anon_id represents the anon_id of the student/team. Anon_ids represents the anon_ids of all the students in the team. Renamed to student_anon_ids for clarity